### PR TITLE
Events handlers

### PR DESCRIPTION
**Pull request by KingsleyIbe**
# In this milestone, I implemented the following:

- Copied the files calculate.js and operate.js into a logic/ directory in my project.
-  Understand what each line of the code does.
- Imported the files in your Calculator component.
- Implemented the event handlers using the math logic from calculate.js and operate.js in my React component.
- Used state and props
-  Passed props to child components
-  Used lifecycle methods, lift state up.

### CODE REVIEW

File: src/Components/Calculator.js [hunk 1]
```diff
@@ -15,30 +27,32 @@ class Calculator extends React.Component {
...
-            <button type="button" className="number" id="0">.</button>
-            <button type="button" className="operator" id="=">=</button>
+            <button type="button" className="operator" id="clear" onClick={this.handleBtnClick}>C</button>
+            <button type="button" className="operator" id="backspace" onClick={this.handleBtnClick}>AC</button>
+            <button type="button" className="operator" id="backspace" onClick={this.handleBtnClick}>+/-</button>
+            <button type="button" className="operator" id="+" onClick={this.handleBtnClick}>&#247;</button>
+            <button type="button" className="operator" id="7" onClick={this.handleBtnClick}>7</button>
+            <button type="button" className="number" id="8" onClick={this.handleBtnClick}>8</button>
+            <button type="button" className="number" id="9" onClick={this.handleBtnClick}>9</button>
+            <button type="button" className="number" id="*" onClick={this.handleBtnClick}>&times;</button>
```
Comment (bimbolabuari) [inline src/Components/Calculator.js]: image

Kindly fix this error, as the web page goes blank when I use the '*' operator sign

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @KingsleyIbe , 

You have done a good job so far 🚀  

## Highlights 🎉 

- Project has an `operate.js` file as required.
- Project has a `calculate.js` file as required.

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

**Comment by KingsleyIbe** [comment]
Hi Abimbola,

Thanks for your feedback
Changes will be effected as recommended.

**Review by D4nielJ — APPROVED** [review]
## Hello @KingsleyIbe, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KingsleyIbe** [comment]
Hi Daniel,

Thank you for reviewing my project

**Pull request merged**